### DISCUSSION TITLE
Error: P1001 Please make sure your database server is running on macOS m2

### DISCUSSION THREAD

**Discussion by hao3304** [prisma/prisma#19829] [Q&A] [answered]
### Bug description

On my macbook m2 or macbook m1, an error was reported when using Prisma to connect to MS SQL, while the same configuration works normally under window.

<img width="813" alt="image" src="https://github.com/prisma/prisma/assets/7592389/88812e91-d15b-47b0-ad67-8b39d5806dcd">

### How to reproduce

1、set .env  
`DATABASE_URL="sqlserver://xx.xx.xx.xx:1433;database=xx;user=sa;password=xxxxxx;encrypt=true;trustServerCertificate=true;"`
2、run command 
`prisma db pull`

### Prisma information

```
generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native"]
}

datasource db {
  provider = "sqlserver"
  url      = env("DATABASE_URL")
}
```

### Environment & setup

- OS: macOS M2
- Database:SQL Server
- Node.js version: v18.16.0

### Prisma Version

```
prisma                  : 4.15.0
@prisma/client          : 4.15.0
Current platform        : darwin-arm64
Query Engine (Node-API) : libquery-engine 8fbc245156db7124f997f4cecdd8d1219e360944 (at node_modules/.pnpm/@prisma+engines@4.15.0/node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Migration Engine        : migration-engine-cli 8fbc245156db7124f997f4cecdd8d1219e360944 (at node_modules/.pnpm/@prisma+engines@4.15.0/node_modules/@prisma/engines/migration-engine-darwin-arm64)
Format Wasm             : @prisma/prisma-fmt-wasm 4.15.0-28.8fbc245156db7124f997f4cecdd8d1219e360944
Default Engines Hash    : 8fbc245156db7124f997f4cecdd8d1219e360944
Studio                  : 0.484.0

```

**Comment by nurul3101** [comment]
Hello @hao3304 👋 

Are you able to connect to this MS SQL Server database - `sqlserver://xx.xx.xx.xx:1433;database=xx;user=sa;password=xxxxxx;encrypt=true;trustServerCertificate=true;` through a database IDE like [TablePlus](https://tableplus.com/) or [DataGrip](https://www.jetbrains.com/datagrip/)?

Basically I just want to verify if database is running at the host that you mentioned.

**Reply by hao3304** [reply]
Hello @nurul3101  
The database is on a remote server, and I have no problem running the same code in window. Then, both DataGrip and Navicat on the Mac can connect. Specifically, the version of MSSQL is 2008
<img width="336" alt="image" src="https://github.com/prisma/prisma/assets/7592389/cd3274d9-4688-4df1-aa88-386df94dbcbd">

**Reply by nurul3101** [reply, marked as answer]
I don't think MSSQL 2008 version is supported. Here's the official list of databases supported by Prisma: [Supported Databases - Prisma](https://www.prisma.io/docs/reference/database-reference/supported-databases)

**Reply by hao3304** [reply]
@nurul3101 Thank you, that should be the problem。
